src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonItem } from '../PersonItem/PersonItem';
+
+type Props = {
+  people: Person[];
+  isError: boolean;
+  isLoading: boolean;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  isError,
+  isLoading,
+}) => {
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {isLoading && <Loader />}
+
+        {isError && !isLoading && (
+          <p data-cy="peopleLoadingError" className="has-text-danger">
+            Something went wrong
+          </p>
+        )}
+
+        {!isError && !isLoading && !people.length && (
+          <p data-cy="noPeopleMessage">There are no people on the server</p>
+        )}
+
+        {!isError && !isLoading && people.length > 0 && (
```

Comment:
'!isLoading' this condition is duplicated, you can create a wrapper for that and add this condition for wrapper